client/src/components/body/lab/GridImages/GridImages.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react'
+import { gridMockData } from '../../../mockData/gridMockData'
+import clsx from "clsx";
+
+const GridImages = () => {
+	// to test this component, go to
+	// localhost:3000/gridimages
+	const [currentFile, setCurrentFile] = useState({})
+	const [id, setId] = useState('')
+	const [active, setActive] = useState(false)
+
+
+
+	const handleGridImage = (e,id) => {
+		e.preventDefault();
+		console.log(e.target,'e')
+		setId(id)
+		const selectImg = gridMockData.filter(img => img.id === id)?.[0]
+		setCurrentFile(selectImg)
+	}
+
+	const handleKeyPress = (e,id) =>{
+		handleGridImage(e,id);
+	}
+
+	const handleEsc = (e) => {
+		if (e.key === "Escape") {
+			setCurrentFile({})
+			setActive(false)
+		};
+	};
+
+	useEffect(() => {
+		if (active) {
+			window.addEventListener("keydown", handleEsc);
+		} else {
+			window.removeEventListener("keydown", handleEsc);
+		}
+		return () => {
+			window.removeEventListener("keydown", handleEsc);
+		};
+	}, [currentFile, active]);
+
+
+
+	const gridImagesClassnames = clsx({
+		"tw-cursor-pointer tw-w-full tw-rounded": true,
+	});
+
+	const gridImageClassnames = clsx({
+		"tw-cursor-pointer": true,
+		'tw-opacity-50': active,
+		"tw-border-double tw-border-8":active
```

Comment:
Active here is a state which is updated by setActive. So it should be on one line, clsx helps with toggling styles so we don't have to do the dirty work inside jsx.

'tw-border-double tw-border-8tw-opacity-50': active,